contrib/cirrus/setup_environment.sh

```diff
@@ -19,6 +19,13 @@ die_unknown() {
     die "Unknown/unsupported \$$var_name '$var_value'"
 }
 
+msg "************************************************************"
+msg "FIXME: force-install catatonit 0.17.0 until CI images are updated"
+msg "************************************************************"
+# FIXME: this is just a temporary workaround to force-install
+# catatonit 0.17.0.  Please remove once the images are updated.
+./hack/install_catatonit.sh --force
```

Comment:
What will this mean for bodhi gating tests? I see catatonit-0.1.6-1 on f34 and f35. Will 0.17.0 be packaged for those, or will this podman version never exist for f34/f35?

Reply:
It's currently in testing for 34/35 -> https://bodhi.fedoraproject.org/updates/?search=catatonit

Reply:
ack, thanks. Guess I better go play with it & give it karma.